12_reading_inputs_and_type_coercion.md

```diff
@@ -156,7 +156,7 @@ A description of what the sequence number represents can be found in Mastering B
     for _ in 0..input_length {
         let txid = read_txid(&mut bytes_slice);
         let output_index = read_u32(&mut bytes_slice);
-        let script = read_script(&mut bytes_slice);
+        let script_sig = read_script(&mut bytes_slice);
```

Comment:
Hm, if we change this here, we'll want to change the Input struct field name as well. Might make sense to remove this from this PR and do this all in a separate PR after merging in your other changes for chapters 12 and 13. I'm happy to do that.

Reply:
I'll do that and open an issue so we remember that later.